packages/messaging/src/controllers/window-controller.test.ts

```diff
@@ -104,6 +108,134 @@ describe('WindowController', () => {
   });
 
   describe('getToken', () => {
+    it('uses default sw if none was registered nor provided', async () => {
+      assert.isUndefined(windowController.getSwReg());
```

Comment:
We don't use `assert` style in tests. Can you please convert `assert`s to `expect`s to be consistent with our convention?